includes/wf_crm_webform_postprocess.inc

```diff
@@ -2367,9 +2371,26 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
           $date = wf_crm_aval($this->data, "$ent:$c:$table:$n:$name", '');
           $val = $date . str_replace(':', '', $val);
         }
+
         // Only known contacts are allowed to empty a field
         if (($val !== '' && $val !== NULL && $val !== []) || !empty($this->existing_contacts[$c])) {
           $this->data[$ent][$c][$table][$n][$name] = $val;
+          if (substr($name, 0, 6) === 'custom') {
+            $multivaluesCreateMode = NULL;
+            // Handle 'Create mode' for multivalues custom fields of Contact entity.
+            if ($ent === 'contact' && is_numeric(substr($name, 7))) {
+              $createModeKey = 'civicrm_' . $c . '_contact_' . $n . '_' . $table . '_createmode';
+              $multivaluesCreateMode = isset($this->data['config']['create_mode'][$createModeKey]) ? (int) $this->data['config']['create_mode'][$createModeKey] : NULL;
+              $multiRecordGid = CRM_Core_BAO_CustomField::isMultiRecordField(substr($name, 7));
```

Comment:
This is a CiviCRM internal function. The goal of this module is to only use the API.
I think you can access this info with something like `wf_crm_get_fields('sets')['cg_' . substr($name, 7)]['max_instances']`